CONTRIBUTING.md

```diff
@@ -25,8 +25,8 @@ gratitude and appreciation of everyone's time and work.
 
 Generally, the first steps are:
 
-- Fork this repository and clone it locally. Note that unlike the git default, our main branch is called `main`.
-- Get a working development environment (see the [README](https://github.com/MoveOnOrg/Spoke/#spoke) and [docs/](https://github.com/MoveOnOrg/Spoke/tree/main/docs))-- reach out through our communication channels (above) if you have issues.
+- Fork this repository and clone it on your local. Our main branch is called `main`.
+- Get a working development environment (see the [ENVIRONMENT](https://github.com/percworld/Spoke/blob/main/docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md) to onboard and [DEVELOPMENT TIPS](https://github.com/MoveOnOrg/Spoke/blob/main/docs/EXPLANATION-development-guidelines.md) which is included in the explanation section [docs/](https://github.com/MoveOnOrg/Spoke/tree/main/docs))-- reach out through our communication channels (above) if you have issues.
```

Comment:
Instead of linking to your fork (https://github.com/percworld/Spoke/blob/main/docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md), can you link it to the main repo? https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md

Reply:
Complete, thank you for noticing this oversight of mine!

Reply:
@percworld Thanks! Did you push the changes? I'm not seeing them in this PR

Reply:
No I have not, I am looking at pulling from #2090 first but not comfortable yet fetching something that has not been merged! Should I go ahead and push without the proper test fixes?

Reply:
Yep, it's fine to push your commit without pulling the test fixes first. I will just test that on my end